crypto/bindings/conversion-proof.ts

```diff
@@ -186,6 +224,7 @@ function proofConversionPerField(
       let prevChallengeScalars = new VecVec(n);
       let prevChallengeCommsMl: MlArray<PolyComm> = [0];
       for (let [, scalars, comms] of prevChallenges) {
+        // TODO is .push() correct if we already initialized to the final length?
```

Comment:
Is there a test to verify it?

Reply:
I now read the Rust docs for `Vec::with_capacity` and it's clear from there that this is correct

Reply:
removing the TODO comments